avocado/utils/linux_modules.py

```diff
@@ -40,7 +40,12 @@
 
 
 def load_module(module_name):
-    # Checks if a module has already been loaded
+    """
+    Checks if a module has already been loaded.
+    :param module_name: Name of module to check
+    :return: True if module is loaded, False otherwise
+    :rtype: Bool
+    """
     if module_is_loaded(module_name):
         return False
```

Comment:
@apahim oh yes you are right but i think this is not my code it was earlier , but no issue i fix this issue raise a PR as soon as possible